app/models/edm/web_resource.rb

```diff
@@ -54,6 +54,30 @@ class WebResource
       application/pdf
     ).freeze
 
+    ALLOWED_FILE_EXTENSIONS = %w(
+      .jpg
+      .jpeg
+      .bmp
+      .gif
+      .png
+      .mp4
+      .webm
+      .mp3
+      .mpeg
+      .pdf
+    ).freeze
+
+    class << self
+      def allowed_extensions
+        ALLOWED_FILE_EXTENSIONS.join(', ')
+      end
+
+      def allowed_mime_types
```

Comment:
Make the constant and method names consistent, i.e. "mime" or "content" in both.